components/admin_console/workspace-optimization/dashboard.scss

```diff
@@ -208,6 +220,21 @@
                     border-top: 1px solid rgba(63, 67, 80, 0.06);
                     background-color: rgb(255, 255, 255);
                 }
+
+                .ctaButtons {
+                    display: flex;
+                    justify-content: flex-start;
+                    margin: 15px 0;
+
+                    button {
+                        max-height: 32px;
+                        padding: 5px 20px !important;
+                        margin-top: 0;
+                        margin-right: 10px;
+                        border-radius: 4px !important;
+                        font-weight: 600;
+                    }
```

Comment:
don't we have common button styles somewhere in the application that we could reuse here?

And are those `!important` really necessary? I would rather increase specificity and don't use `!important`

Reply:
> don't we have common button styles somewhere in the application that we could reuse here?
yes, the buttons already have the majority of the css styles set as an inline class from a button-class that is used across the system console that looks like the one in the design. These styles are just to make them fit with the design.

> And are those !important really necessary? I would rather increase specificity and don't use !important
Yes, they are important, and about being more specific, this class is applied as a class module, so the next specificity level would be to put the style inline with the button tag, which I don't consider necessary. I know important is not good but in this case it is a necessary evil 👿